shipwright/build-example.yaml

```diff
@@ -0,0 +1,33 @@
+---
+apiVersion: shipwright.io/v1alpha1
+kind: Build
+metadata:
+  name: multiarch-native-buildah-ex
+  namespace: test2
+spec:
+  source:
+    url: https://github.com/shipwright-io/sample-go
+    contextDir: docker-build
+  strategy:
+    name: multi-arch-native-buildah
+    kind: ClusterBuildStrategy
+  paramValues:
+    - name: architectures
+      values:
+        - value: "arm64"
+        - value: "amd64"
+    - name: build-contexts # In place of the `as` array in build configs
+      values:
+        - value: "ghcr.io/shipwright-io/shipwright-samples/golang:1.18=docker://quay.io/aleskandrox/my-shipwright-built-image:golang-1.19"
```

Comment:
Is this equivalent to golang-builder we're using?

Reply:
No that is just an example I made to expose how the current buildconfigs should be converted into Shipwright Builds. And it is the same example as upstream (just a  hello-world), where I pinpointed for us how to replace the images, or set the requests/limits, etc...

Reply:
Should I clarify this better in the yaml itself or just keep it out of the repo and just start converting the buildconfigs?

Reply:
Ah, okay, example is fine at this stage